font/src/darwin/mod.rs

```diff
@@ -259,6 +313,32 @@ pub fn get_family_names() -> Vec<String> {
     owned_names
 }
 
+
+/// Return fallback descriptors for font/language list
+fn cascade_list_for_languages(
+    ct_font: &CTFont,
+    languages: &Vec<String>
+) -> Vec<Descriptor> {
+
+    let lang:Vec<CFString> = languages.clone().into_iter()
```

Comment:
Can this just be `languages.iter()`? Looks like the strings are just getting borrowed in the `map` anyway.